conan/cli/commands/lock.py

```diff
@@ -8,6 +9,9 @@
 from conan.cli.printers.graph import print_graph_packages, print_graph_basic
 from conan.internal.model.lockfile import Lockfile, LOCKFILE
 from conan.internal.model.recipe_ref import RecipeReference
+from conan.errors import ConanException
+from conan.internal.model.version import Version
+from conan.internal.model.version_range import VersionRange
```

Comment:
We have learned these are problematic, and we are moving away of the ``cli`` layer importing anything else besides ``cli`` and ``api`` layers. So the best would be to put the required functionality directly into the api layer